src/auth/jwt.strategy.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable, UnauthorizedException } from '@nestjs/common';
+import { PassportStrategy } from '@nestjs/passport';
+import { ExtractJwt, Strategy } from 'passport-jwt';
+import { Env } from 'src/env/index';
+import { JwtPayload } from './jwt-interface';
+import { RedisService } from '@liaoliaots/nestjs-redis';
+
+@Injectable()
+export class JwtStrategy extends PassportStrategy(Strategy, 'jwt') {
+  constructor(
+    private readonly redisService: RedisService,
```

Comment:
PrimathonAICodeReviewer

Consider adding error handling for Redis service operations.
```diff
+     try {
+         const client = this.redisService.getClient();
+     } catch (error) {
+         // Handle error, e.g., log or throw custom exception
+     }
```



<!-- This is an auto-generated comment by OSS PrimathonAICodeReviewer -->